plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java

```diff
@@ -325,18 +325,26 @@ protected void configure(Mapping config, T instance, boolean dryrun, Configurati
                 if (attribute.isMultiple()) {
                     List<Object> values = new ArrayList<>();
                     for (CNode o : sub.asSequence()) {
-                        Object value =
-                                dryrun ?
-                                        configurator.check(o, context):
-                                        configurator.configure(o, context);
-                        values.add(value);
+                        try{
+                            Object value =
+                                    dryrun ?
+                                            configurator.check(o, context):
+                                            configurator.configure(o, context);
+                            values.add(value);
+                        } catch (Exception ex) {
+                            throw new ConfiguratorException(configurator, "configuration is invalid", ex);
```

Comment:
You should now throw but collect exceptions.

Reply:
yeah, I saw my solution wasn't the best one. But I'm not sure what you mean about collecting exceptions.